### PR TITLE
data:akhil3shh

### PR DESCRIPTION
**Pull request by akhil3shh**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/3450"></a>

### CODE REVIEW

**Review by Pradumnasaraf — APPROVED** [review]

File: data/akhil3shh.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    {
+      "name": "Instagram",
+      "url": "https://www.instagram.com/akhileshhh.exe/",
+      "icon": "FaInstagram"
+    },
+    {
+      "name": "GitHub",
+      "url": "https://github.com/akhil3shh",
+      "icon": "FaGithub"
+    },
```
Comment (Pradumnasaraf) [inline data/akhil3shh.json:27]:
```suggestion
    }
```

**Review by Pradumnasaraf — CHANGES_REQUESTED** [review]
Hey,

There is an extra `,` in the JSON file.Please remove that

**Review by Panquesito7 — APPROVED** [review]
Thanks! 🎉

**Review by EmmaDawsonDev — APPROVED** [review]
Looks good 👍

**Pull request merged**